third-party/flatbuffers/CMakeLists.txt

```diff
@@ -18,4 +18,14 @@ therock_cmake_subproject_declare(therock-flatbuffers
 therock_cmake_subproject_provide_package(therock-flatbuffers flatbuffers lib/cmake)
 therock_cmake_subproject_activate(therock-flatbuffers)
 
+if(THEROCK_ENABLE_HIPDNN)
+  therock_provide_artifact(flatbuffers
+    DESCRIPTOR artifact-flatbuffers.toml
```

Comment:
does `TARGET_NEUTRAL` apply here?

see cmake code usage for definition in `cmake/therock_artifacts.cmake`
```
  # Determine top-level name.
  if(ARG_TARGET_NEUTRAL)
    set(_bundle_suffix "_generic")
  else()
    set(_bundle_suffix "_${THEROCK_AMDGPU_DIST_BUNDLE_NAME}")
  endif()
```